packages/formation/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@department-of-veterans-affairs/formation",
-  "version": "7.0.2",
+  "version": "7.0.3",
```

Comment:
@bkjohnson I'm assuming this needs a version bump, correct?

Reply:
Yeah, that's correct. [Here are the instructions](https://department-of-veterans-affairs.github.io/veteran-facing-services-tools/getting-started/common-tasks/updating-formation/) for how to update & publish. I believe step 4 "Create a release" doesn't work, but you can still manually create a github release.